app/Shell/DockerTags.php

```diff
@@ -61,6 +64,54 @@ public function getTags(): Collection
         return $sortedTags->values()->filter();
     }
 
+    /**
+     * Filter that ensures images that do not support arm architecture are filtered out.
+     *
+     * @return callable
+     */
+    protected function armSupportedImagesOnlyFilter()
+    {
+        return function ($tags) {
+            $platform = $this->platform();
+
+            // We only want tags that specify arm64 in the supported architecture.
+
+            return $tags->filter(function ($tag) use ($platform) {
+                return collect($tag['images'])
+                    ->pluck('architecture')
+                    ->contains($platform);
+            });
```

Comment:
```suggestion
            // Filter to only tags that list "arm64" in the supported architecture
            return $tags->filter(function ($tag) use ($platform) {
                return collect($tag['images'])
                    ->pluck('architecture')
                    ->contains($this->platform());
            });
```

Reply:
Also... we're directly calling "arm64" in the next function, but using `$this->platform()` here to refer to `arm64`. Obviously in the next function `arm64` won't actually be the result of `platform()`, but I'm still inclined just to use a string in both cases. So maybe instead of `->contains($this->platform())`, let's just do `->contains('arm64')`?

Reply:
Done!